src/Main.js

```diff
@@ -39,7 +39,7 @@ export { CAMERA_TYPE } from 'Renderer/Camera';
 
 // Internal itowns format
 export { default as Feature, FeatureCollection, FeatureGeometry, FEATURE_TYPES } from 'Core/Feature';
-export { default as Style } from 'Core/Style';
+// export { default as Style } from 'Core/Style';
```

Comment:
why do you need this change ? I think it is out of scope of this PR and we should uncomment it for now.

Reply:
Indeed. It's not in the scope of the PR and was just a test to see if it can be removed safely in the futur (with the actual modification).